fuel/transformers/__init__.py

```diff
@@ -258,6 +274,30 @@ def get_data(self, request=None):
         return tuple(numpy.asarray(source_data) for source_data in data)
 
 
+class Unpack(Transformer):
+    """Unpacks batches to compose a stream of examples.
+    This class is the inverse of the Batch class: it turns a minibatch into
+    a stream of examples.
+    Parameters
+    ----------
+    data_stream : :class:`AbstractDataStream` instance
+        The data stream to unpack
+    """
+    def __init__(self, data_stream, batch=False):
```

Comment:
No need to add it to the signature here either, because it won't make sense for the user to pass `batch=True`